site2/docs/reference-pulsar-admin.md

```diff
@@ -2369,20 +2348,279 @@ Remove the message TTL for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics get-retention tenant/namespace/topic
+```
+
+### `set-retention`
+Set the retention policy that is applied to a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-retention tenant/namespace/topic options
+```
+
+### `remove-retention`
+Remove the retention policy that is applied to a topic.
+
+Usage
+```bash
+$ pulsar-admin topics remove-retention tenant/namespace/topic
+```
+
+### `get-dispatch-rate`
+Get the message dispatch rate for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-dispatch-rate tenant/namespace/topic
+```
+
+### `set-dispatch-rate`
+Set the message dispatch rate for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-dispatch-rate tenant/namespace/topic options
+```
+
+### `remove-dispatch-rate`
+Remove the message dispatch rate for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics remove-dispatch-rate tenant/namespace/topic
+```
+
+### `get-max-unacked-messages-per-subscription`
+Get the maximum number of unacked messages per subscription for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics get-max-unacked-messages-per-subscription tenant/namespace/topic
+```
+
+### `set-max-unacked-messages-per-subscription`
+Set the maximum number of unacked messages per subscription for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-max-unacked-messages-per-subscription tenant/namespace/topic options
+```
+
+### `remove-max-unacked-messages-per-subscription`
+Remove the maximum number of unacked messages per subscription for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics remove-max-unacked-messages-per-subscription tenant/namespace/topic
+```
+
+### `get-max-unacked-messages-per-consumer`
+Get the maximum number of unacked messages per consumer for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics get-max-unacked-messages-per-consumer tenant/namespace/topic
+```
+
+### `set-max-unacked-messages-per-consumer`
+Set the maximum number of unacked messages per consumer for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-max-unacked-messages-per-consumer tenant/namespace/topic options
+```
+
+### `remove-max-unacked-messages-per-consumer`
+Remove the maximum number of unacked messages per consumer for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics remove-max-unacked-messages-per-consumer tenant/namespace/topic
+```
+
+### `get-delayed-delivery`
+Get the delayed delivery policy for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics get-delayed-delivery tenant/namespace/topic
+```
+
+### `set-delayed-delivery`
+Set the delayed delivery policy for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-delayed-delivery tenant/namespace/topic options
+```
+
+### `remove-delayed-delivery`
+Remove the delayed delivery policy for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics remove-delayed-delivery tenant/namespace/topic
+```
+
+### `get-max-producers`
+Get the maximum number of Producers for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics get-maxProducers tenant/namespace/topic
```

Comment:
the title should be consistent with the command: get-maxProducers, same comments for the following.

Reply:
The command `get-maxProducers` is deprecated. See the screenshot above.